backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -493,7 +493,28 @@ public async System.Threading.Tasks.Task<Project> UpdateAsync(Project project)
             {
                 originalProject.Note = project.Note;
                 originalProject.PublicNote = project.PublicNote;
-                this.Context.SaveChanges();
+                try
```

Comment:
regrettably this is incorrect.  A concurrency error was correctly being thrown by the API.  The frontend however wasn't correctly handling an update and then subsequent updates were using an old `RowVersion`.